shopinvader_search_engine/models/ir_exports_line.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
+# Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IrExportsLine(models.Model):
+    _inherit = "ir.exports.line"
+
+    # Allow to turn on/off index data for products and categories
+    active = fields.Boolean(string="Active", default=True)
```

Comment:
"ir.exports.line" don't have views? how they will enable/disable it?

Reply:
nope, they don't. ATM this is just for technical usage (disable things via custom modules).

Reply:
FTR moved here https://github.com/OCA/server-tools/pull/1904